ImgBot.Function/CommitMessage.cs

```diff
@@ -24,7 +24,7 @@ public static string Create(Dictionary<string, Tuple<double, double>> optimizedI
             foreach (var optimizedImage in optimizedImages.Keys)
             {
                 var percent = (1 - (optimizedImages[optimizedImage].Item2 / optimizedImages[optimizedImage].Item1)) * 100;
-                imageLog.AppendFormat("{0} -- {1:N2}kb -> {2:N2}kb ({3:0.##}%)", optimizedImage, optimizedImages[optimizedImage].Item1, optimizedImages[optimizedImage].Item2, percent);
+                imageLog.AppendFormat("{0} -- {1:N2}kb -> {2:N2}kb ({3:0.##}%)", optimizedImage.Replace('\\', '/'), optimizedImages[optimizedImage].Item1, optimizedImages[optimizedImage].Item2, percent);

```

Comment:
Maybe u could handle this in the `FileName` property of the `CompressionResult` class from the other PR?

Reply:
Yeah that makes sense - I did this one first.
I think I'll make all the formatting a method on CompressionResult.cs